docs/index.rst

```diff
@@ -1,14 +1,38 @@
-###########################
-Qiskit Nature Documentation
-###########################
+######################
+Qiskit Nature overview
+######################
+
+Overview
+==============
+
+**Qiskit Nature** is an open-source framework that supports problems including ground state energy computations,
+excited states and dipole moments of molecule, both open and closed-shell.
+
+The code comprises chemistry drivers, which when provided with a molecular
+configuration will return one and two-body integrals as well as other data that is efficiently
+computed classically. This output data from a driver can then be used as input in Qiskit
+Nature that contains logic which is able to translate this into a form that is suitable
+for quantum algorithms. The conversion first creates a FermionicOperator which must then be mapped,
+e.g. by a Jordan Wigner mapping, to a qubit operator in readiness for the quantum computation.
```

Comment:
```suggestion
for quantum algorithms. The conversion first creates a fermionic operator which must then be mapped,
e.g. by a Jordan Wigner mapping, to a qubit operator in readiness for the quantum computation.
```

This aligns better with the qubit operator mentioned later. If we don't want this, we should say `FermionicOp`, matching the current name of the class.